nautobot/docs/release-notes/index.md

```diff
@@ -1,18 +1,14 @@
 # Release Notes
 
-## [2.4 - December 2024](version-2.4.md)
+## [2.4 - January 2025](version-2.4.md)
 
-- Apps Marketplace Page and Installed Apps Page Tile View
+- Virtual Device Context Data Models
+- Wireless Data Models
+- Apps Marketplace Page
 - Event Publication Framework
 - Jinja2 Template Rendering REST API
-- Job `success` Log Level
 - Kubernetes Job Execution and Job Queue Data Model
-- Per-User Time Zone Support
-- REST API `exclude_m2m` Support
-- Singleton Jobs
```

Comment:
Not "significant" enough to include? Or why were these removed?

Reply:
Trying to keep the high level index focused on the marquee features. To that end, we do have formality within OCTO/Marketing as to what those are, so there is alignment with what the marketing plan is.